client/src/components/Onboarding/Availability/Availability.tsx

```diff
@@ -18,11 +21,12 @@ interface Props {
     }: FormikHelpers<{
       hours: string;
       days: string;
-    }>
+    }>,
   ) => void;
 }
 
 const Availability = ({ handleSubmit }: Props): JSX.Element => {
+  const classes = useStyles();
   const [hours, setHours] = useState({ start: '9:00', end: '17:00' });
   const [days, setDays] = useState({
     Sunday: false,
```

Comment:
Do we have to use state here? Would Formik be able to handle that for us as opposed to tracking it here?

Reply:
Can you explain further? I have visited Formik doc's on handling selects and checkboxes but so far just a few red lines about ...props, ...children.